jobs/build_ova/ova_post_test.groovy

```diff
@@ -10,18 +10,27 @@ node(){
             checkout scm
             def function_test = load("jobs/FunctionTest/FunctionTest.groovy")
             def repo_dir = pwd()
-            withCredentials([
-                usernamePassword(credentialsId: 'OVA_CREDS', 
-                                    passwordVariable: 'OVA_PASSWORD', 
-                                    usernameVariable: 'OVA_USER'),
-                string(credentialsId: 'vCenter_IP', variable: 'VCENTER_IP'), 
-                string(credentialsId: 'Deployed_OVA_INTERNAL_IP', variable: 'OVA_INTERNAL_IP')
-                ]) {
-                // Start to run test
-                def TESTS = "${env.OVA_POST_TESTS}"
-                def OVA_STASH_NAME = "${env.OVA_STASH_NAME}"
-                def OVA_STASH_PATH = "${env.OVA_PATH}"
-                function_test.ovaPostTest(TESTS, OVA_STASH_NAME, OVA_STASH_PATH, repo_dir)
+            try{
+                withCredentials([
+                    usernamePassword(credentialsId: 'OVA_CREDS', 
+                                        passwordVariable: 'OVA_PASSWORD', 
+                                        usernameVariable: 'OVA_USER'),
+                    string(credentialsId: 'vCenter_IP', variable: 'VCENTER_IP'), 
+                    string(credentialsId: 'Deployed_OVA_INTERNAL_IP', variable: 'OVA_INTERNAL_IP')
+                    ]) {
+                    // Start to run test
+                    def TESTS = "${env.OVA_POST_TESTS}"
+                    def OVA_STASH_NAME = "${env.OVA_STASH_NAME}"
+                    def OVA_STASH_PATH = "${env.OVA_PATH}"
+                    function_test.ovaPostTest(TESTS, OVA_STASH_NAME, OVA_STASH_PATH, repo_dir)
+                }
+            }catch(error){
+                echo "Caught: ${error}"    
+                currentBuild.result = "FAILURE"
+            }finally{
+                function_test.archiveArtifactsToTarget("OVA_POST_TEST")
+                def shareMethod = load("jobs/ShareMethod.groovy")
+                shareMethod.sendResult(true,true)
```

Comment:
Could you please paste a pipeline view of test MasterCI?
There is already ```sendResult``` in MasterCI.
I'm not sure whether there will be 2 stage "send result"...